setuptools_rust/extension.py

```diff
@@ -240,7 +240,17 @@ def _metadata(self) -> "_CargoMetadata":
             ]
             if self.cargo_manifest_args:
                 metadata_command.extend(self.cargo_manifest_args)
-            self._cargo_metadata = json.loads(subprocess.check_output(metadata_command))
+
+            env = os.environ.copy()
+            try:
+                payload = subprocess.check_output(
+                    metadata_command, env=env, encoding="latin-1"
```

Comment:
The default is to inherit the current environment? I don't think `env=env` is necessary?

Reply:
Fair point, I just wanted to make sure `env` definitely is present, but will remove.